service/lib/agama/dbus/software/product.rb

```diff
@@ -151,6 +151,27 @@ def registered_addons
           end
         end
 
+        # list of available addons
+        #
+        # @return [Array<Hash<String, Object>>] List of addons
+        def available_addons
+          addons = backend.registration.available_addons || []
+
+          addons.map do |a|
```

Comment:
Unfortunately this fails with DBus error:  `Specified type has 2 members but value has 9 members; caused by 1 sender=:1.1 -> dest=org.opensuse.Agama.Software1...` 

I guess I need to wrap something into an Array... @mvidner ?